### PR TITLE
[11.0][ADD] account_invoice_pricelist_discount

### PR DESCRIPTION
**Pull request by Bearnard21**
module added

### CODE REVIEW

**Review by aayartsev — APPROVED** [review]
Looks good

File: account_invoice_pricelist_discount/models/account_invoice.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            quantity=self.quantity,
+            date_order=self.invoice_id.date_invoice,
+            pricelist=pricelist.id,
+            uom=self.uom_id.id,
+            fiscal_position=(
+                self.invoice_id.partner_id.property_account_position_id.id)
+        )
+
+        if pricelist.discount_policy == 'without_discount':
+            if product.list_price != 0:
```
Comment (ivs-cetmix) [inline account_invoice_pricelist_discount/models/account_invoice.py]: What happens if product.list_price == 0?
If there is no action for this condition is involved then why not to check this value in the previous "if" at L28?

File: account_invoice_pricelist_discount/tests/test_account_invoice_pricelist.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+                    'product_id': self.product.product_variant_ids[:2].id,
+                    'name': 'Test line',
+                    'quantity': 1.0,
+                    'price_unit': 100.00,
+                })
+            ],
+        })
+
+    def test_account_invoice_change_pricelist(self):
+        """Check discount applied not shown with """
```
Comment (ivs-cetmix) [inline account_invoice_pricelist_discount/tests/test_account_invoice_pricelist.py]: What does it mean? Docstring is not clear

File: account_invoice_pricelist_discount/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1,5 @@
+This module extends OCA account_invoice_pricelist module behaviour
+
+It show public price & discount to the customer in invoice lines according to the the pricelist selected.
```
Comment (ivs-cetmix) [inline account_invoice_pricelist_discount/readme/DESCRIPTION.rst:3]: It adds discount directly into the "discount" filed of the  invoice lines based on the the pricelist selected

**Review by ivs-cetmix — CHANGES_REQUESTED** [review]

File: account_invoice_pricelist_discount/tests/test_account_invoice_pricelist.py (hunk 2 above)
Comment (Bearnard21) [reply account_invoice_pricelist_discount/tests/test_account_invoice_pricelist.py]: fixed

File: account_invoice_pricelist_discount/models/account_invoice.py (hunk 1 above)
Comment (Bearnard21) [reply account_invoice_pricelist_discount/models/account_invoice.py]: fixed

File: account_invoice_pricelist_discount/readme/DESCRIPTION.rst (hunk 3 above)
Comment (Bearnard21) [reply account_invoice_pricelist_discount/readme/DESCRIPTION.rst:3]: Fixed

**Review by ivs-cetmix — APPROVED** [review]
Functional OK

**Review by aayartsev — APPROVED** [review]
Looks good

**Pull request merged**